hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1500,6 +1534,55 @@ components:
           $ref: "#/components/schemas/NftAllowances"
         links:
           $ref: "#/components/schemas/Links"
+    OpcodesResponse:
+      type: object
+      properties:
+        address:
```

Please add the same to all hex encoded fields.

```suggestion
        address:
          format: binary
```